docs/Measuring_Resonances.md

```diff
@@ -51,6 +51,13 @@ Note that on a bed slinger printer one must design 2 mounts: one for the
 toolhead and one for the bed, and run the measurements twice. See the
 corresponding [section](#bed-slinger-printers) for more details.
 
+**Attention:** neither the accelerometer nor the screws that hold it in
+place (if any) should not touch any metal parts of the printer.
```

Comment:
Double negative?  How about: `make sure the accelerometer and any screws that hold it in
place do not touch any metal parts of the printer.`

Reply:
Yes, that indeed sounds better.